graphql/src/adapters/graphql/dataTypes/storage_metadata.py

```diff
@@ -0,0 +1,10 @@
+# noinspection PyPackageRequirements
+import strawberry
+
+from src.adapters.graphql.dataTypes.common import InternalServerErrorStrawberryType
+from src.entities.storage_metadata import StorageSchemaVersion
+
+GetStorageSchemaVersionStrawberryResponse = strawberry.union(
```

Comment:
It is a graphql-specific layer, and should be marked as such, if only for code clarity, as having classes with the same name is to be avoided. Switched to `GraphqlType` to avoid tech-specific naming.